pkgs/tools/X11/xpra/default.nix

```diff
@@ -84,6 +86,12 @@ in buildPythonApplication rec {
     ./fix-122159.patch # https://github.com/NixOS/nixpkgs/issues/122159
   ];
 
+  postPatch = lib.optionalString stdenv.isLinux ''
+    # Note: xposix is renamed to posix in v5.
+    substituteInPlace xpra/platform/xposix/features.py \
+      --replace "/usr/bin/xdg-open" "${xdg-utils}/bin/xdg-open"
```

Comment:
--replace-quiet possibly?

https://nixos.org/manual/nixpkgs/stable/#fun-substitute-replace-quiet

Reply:
we should use `--replace-fail`; if it fails, we want to know.

Reply:
Fair enough! :+1: